nomad/leader.go

```diff
@@ -62,6 +62,14 @@ var minACLRoleVersion = version.Must(version.NewVersion("1.4.0"))
 // 1.5, otherwise it's hard to test the functionality
 var minACLAuthMethodVersion = version.Must(version.NewVersion("1.4.3-dev"))
 
+// minACLBindingRuleVersion is the Nomad version at which the ACL binding rules
+// table was introduced. It forms the minimum version all federated servers
+// must meet before the feature can be used.
+//
+// TODO: version constraint will be updated for every beta or rc until we reach
+// 1.5, otherwise it's hard to test the functionality
+var minACLBindingRuleVersion = version.Must(version.NewVersion("1.4.3-dev"))
```

Comment:
Once we go to 1.5-beta we can update this to `1.5.0` and then use [`Version.Core`](https://pkg.go.dev/github.com/hashicorp/go-version?#Version.Core) on the version we check.